_includes/header.html

```diff
@@ -1,217 +1,181 @@
+    </button>
+    <div class="syn-visible-lg">
+      <a href="https://algorithmia.com/mlops" class="syn-link-no-decoration">
+        <button class="syn-btn text">MLOps</button>
+      </a> 
+      <a href="https://algorithmia.com/product" class="syn-link-no-decoration">
+        <button class="syn-btn text syn-ml-8">Product</button>
+      </a>
+      <a href="https://algorithmia.com/pricing" class="syn-link-no-decoration">
+        <button class="syn-btn text syn-ml-8">Pricing</button>
+      </a>
+      <div class="syn-dropdown syn-ml-8">
+        <button class="syn-btn text" onclick="this.parentNode.classList.toggle('open')">
+          Learn
+          <svg width="7px" height="4px" viewBox="0 0 7 4" class="syn-ml-4">
+            <polygon fill="currentColor" points="0 0 6.688 0 3.344 3.344"></polygon>
+          </svg>
+        </button>
+        <div class="open-break syn-bg-black syn-mt-8"></div>
+        <div class="syn-dropdown-list" onclick="this.parentNode.classList.toggle('open')">
+          <div class="syn-dropdown-list-item syn-body-2 syn-mb-0">
+                  <a href="https://algorithmia.com/blog" class="syn-ph-48 syn-pv-16">
+                    <div>
+                      <div class="syn-font-weight-bold">Blog</div>
+                      <div class="link-description">
+                        Special topics in machine learning
+                      </div>
+                    </div>
+                  </a>
+                </div>
+                <div class="syn-dropdown-list-item syn-body-2 syn-mb-0">
+                  <a href="https://algorithmia.com/resources" class="syn-ph-48 syn-pv-16">
+                    <div>
+                      <div class="syn-font-weight-bold">Resources</div>
+                      <div class="link-description">
+                        Case studies, videos, and reports
+                      </div>
+                    </div>
+                  </a>
+                </div>
+                <div class="syn-dropdown-list-item syn-body-2 syn-mb-0">
+                  <a href="https://algorithmia.com/developers" class="syn-ph-48 syn-pv-16">
+                    <div>
+                      <div class="syn-font-weight-bold">Docs</div>
+                      <div class="link-description">
+                        Platform technical documentation
+                      </div>
+                    </div>
+                  </a>
+                </div>
+                <div class="syn-dropdown-list-item syn-body-2 syn-mb-0">
+                  <a href="https://algorithmia.com/events" class="syn-ph-48 syn-pv-16">
+                    <div>
+                      <div class="syn-font-weight-bold">Events</div>
+                      <div class="link-description">
+                        Webinars, talks, and trade shows
+                      </div>
+                    </div>
```

Comment:
Each one of the anchors in this list has an extra wrapper div. Are these necessary?